app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -11,13 +11,13 @@ class OmniauthCallbacksController < Devise::OmniauthCallbacksController
 
     def handle(provider)
       auth_hash = request.env['omniauth.auth']
-      uid = auth_hash[:uid]
+      username = auth_hash.info.email.split("@")[0]
```

Comment:
Use single quotes here, it's a styling thing, and the reason why linters have failed

Reply:
Sure @differentreality